templates/zerver/help/add-custom-emoji.md

```diff
@@ -0,0 +1,59 @@
+# Add custom emoji
+
+If you are an administrator of a Zulip organization, you can add custom emojis
+to your organization for your organization members to use.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
+of the right sidebar.
+
+2. Select **Administration** from the dropdown menu that appears.
+
+    ![Administration dropdown](/static/images/help/administration.png)
+
+3. Upon clicking **Administration**, your view will be replaced with the
+**Administration** page. You will be located on the **Organization settings**
+tab by default.
+
+    ![Administration page](/static/images/help/admin-window.png)
+
+4. Scroll down to the **Custom emoji** section located beneath the
+**Organization settings** section.
+
+    ![Custom emoji](/static/images/help/custom-emoji.png)
+
+5. In the green section labeled **Add a New Emoji**, find the **Emoji name** and
+**Emoji URL** fields.
+
+    * In the **Emoji name** field, enter a phrase for your emoji. No spaces may
```

Comment:
I think it would be great if we can specify which letters are acceptable in emoji names.